modules/oc-mirror-imageset-config-params.adoc

```diff
@@ -0,0 +1,212 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-disconnected.adoc
+
+[id="oc-mirror-imageset-config-params_{context}"]
+= Image set mirroring configuration parameters
+
+// TODO intro
+
+// TODO: Updated all of the "mappings" to "String" like elsewhere in our doc - okay?
+// TODO: removed "Kind" row - okay with that?
+
+.`ImageSetConfiguration` parameters
+[cols="2,2,1a",options="header"]
+|===
+|Parameter
+|Description
+|Values
+
+|`apiVersion`
+|The API version for the `ImageSetConfiguration` content.
+|String. For example: `mirror.openshift.io/v1alpha1`.
+
+// TODO: Changed example (lowered the "v") - confirm
```

Comment:
@afflom @jpower432 The Google doc had `V1alpha1` as the value for apiVersion, and I updated it to this. Including lowering the "V". Just checking that this is right.

Reply:
@bergerhoffer Yep! You have it right.